Scripts/validate-branch-name.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+LC_ALL=C
+
+local_branch="$(git rev-parse --abbrev-ref HEAD)"
+
+valid_branch_regex="^(feature|fix|chore|improvement|release)\/[a-z0-9._-]+$"
+
+message="Invalid branch name. Branch names should follow this format: $valid_branch_regex. Your commit will be rejected. Rename your branch to a valid name and try again."
```

Comment:
Is this helpful enough? Maybe an example of a valid branch name would do better

Reply:
Looks simple enough to me, do you think it's not?

Reply:
Updated anyway, better?

Reply:
I am not really good in reading regex expressions :) Leave it as is if it's really clear to everyone

Reply:
🙇